src/components/DiagramFrame/SeqDiagram/LifeLineLayer/Participant.vue

```diff
@@ -52,9 +52,10 @@ export default {
     const translate = computed(() => {
       let top = intersectionTop.value + scrollTop.value
       if (intersectionTop.value > INTERSECTION_OFFSET && store?.state.stickyOffset) top += store?.state.stickyOffset
-      if (top < props.offsetTop + PARTICIPANT_TOP) return 0
-      const diagramHeight = store?.state.diagramElement.clientHeight || 0
-      return Math.min(top - PARTICIPANT_TOP, diagramHeight - PARTICIPANT_HEIGHT) - props.offsetTop
+      const diagramHeight = store?.state.diagramElement?.clientHeight || 0
```

Comment:
consider map `diagramElement` to computed property with vuex `mapState`?

Reply:
I think it is not necessary, as we only need this value when we the page is scrolling